SGoopas/Assets/Scripts/Game/Spotlights/Physics2DApplicator.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEngine;
 
 
-public class Physics2DApplicator : ShadowApplicator
+public class Physics2DApplicator : MonoBehaviour, ShadowApplicator
```

Comment:
Yeah, as Danny mentioned, this will break the spotlight code, since you can't instantiate a monobehaviour without it being attached to a game object, and Physics2DApplicator isn't intended to be attached to a game object itself